english/find-exit/leave.md

```diff
@@ -6,9 +6,11 @@ There are two paths outside. One leads into a forest. One leads into the mountai
 
 In the distance, you notice a tower that has caught fire, but is smoking a deep shade of purple.
 
+You then notice that the purple haze is that of the burning mushrooms in the distance. You start to feel disoriented. 
+
 Overhead, you hear a loud, running noise. It's a chococopter!
 
-You have a slight cold.  You might want to call in sick to work. What do you do?
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.